foxy/ci-nightly-performance.yaml

```
@@ -105,6 +105,11 @@ targets:
 type: ci-build
 underlay_from_ci_jobs:
 - nightly-extra-rmw-release
+archive_files:
+  - ws/test_results/buildfarm_perf_tests/performance_test_results_*.csv
+  - ws/test_results/buildfarm_perf_tests/overhead_test_results_*.csv
+  - ws/test_results/buildfarm_perf_tests/overhead_node_test_results_*.csv
+  - ws/test_results/buildfarm_perf_tests/performance_test_two_process_results_*.csv
```

For consistency, this YAML file isn't indenting for lists.

```suggestion
- ws/test_results/buildfarm_perf_tests/performance_test_results_*.csv
- ws/test_results/buildfarm_perf_tests/overhead_test_results_*.csv
- ws/test_results/buildfarm_perf_tests/overhead_node_test_results_*.csv
- ws/test_results/buildfarm_perf_tests/performance_test_two_process_results_*.csv
```

---

Good point, see 216f99ac9872f0dbed497629eaeec9829d5f1ea0.